packages/bsky/src/services/indexing/processor.ts

```diff
@@ -84,7 +84,7 @@ export class RecordProcessor<T, S> {
     if (inserted) {
       this.aggregateOnCommit(inserted)
       await this.handleNotifs({ inserted })
-      return
+      return this.params.notifsForInsert(inserted)
```

Comment:
we may want to return something else here that has more details on the record - so that the notification server doesn't have to do a bunch of re-querying stuff out of the db